npm/pkg/dataplane/policies/policy.go

```diff
@@ -183,9 +183,12 @@ const (
 // Possible MatchTypes.
 // MatchTypes with 2 locations (e.g. DstDst) are for ip and port respectively.
 const (
-	SrcMatch    MatchType = 0
-	DstMatch    MatchType = 1
+	SrcMatch MatchType = 0
```

Comment:
I am fine if you have ideas to better code, but do we have `matchTypeStrings` map just below this code?
- https://github.com/Azure/azure-container-networking/blob/master/npm/pkg/dataplane/policies/policy.go#L191
Always glad if you can make better one. If so, please make different PR for them. I think it used in several place and easy to have one simple PR.

@huntergregory  There is one question about `DstDstMatch` (https://github.com/Azure/azure-container-networking/blob/master/npm/pkg/dataplane/policies/policy.go#L188)
```go
// TODO(jungukcho) Question - Why it is 3, not 2?
DstDstMatch MatchType = 3
```

Reply:
I had all the combinations before (SrcSrcMatch was 2), so it's leftover from that.

Reply:
I agree, since there's a map already, this should be a separate PR change

Reply:
If so, I will revise it to below code. 
```go
// Possible MatchTypes.
const (
	SrcMatch MatchType = 0
	DstMatch MatchType = 1
	// MatchTypes with 2 locations (e.g. DstDst) are for ip and port respectively.
	DstDstMatch MatchType = 2
	// This is used for podSelector under spec. It can be Src or Dst based on existence of ingress or egress rule.
	EitherMatch MatchType = 3
)
```